frontend/src/views/admin/WaitingListPanelView.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+  <div class="flex w-full justify-center">
+    <alert-box
+      @close="pageNotification = ''"
+      v-if="pageNotification"
+      :scheme="alertSchemes.success"
+    >
+      {{ pageNotification }}
+    </alert-box>
+    <alert-box
+      @close="pageError = ''"
+      v-if="pageError"
+    >
+      {{ pageError }}
+    </alert-box>
+  </div>
+  <admin-nav/>
+  <div v-if="displayPage">
+    <data-table
+      data-name="Waiting List Users"
+      :allow-multi-select="false"
+      :data-list="filteredUsers"
+      :columns="columns"
+      :filters="filters"
+      :loading="loading"
+    >
+    </data-table>
+  </div>
+  <div v-else class="flex size-full min-h-[75vh] items-center justify-center">
+    <loading-spinner/>
+  </div>
+</template>
+
+<script setup>
```

Comment:
Do we want to put the script tag at the top for all (new) single file components?

Reply:
Sure! I think that's probably a good plan.